scripts/update.sh

```diff
@@ -9,7 +9,8 @@ ACCESS_TOKEN=""
 
 OXA_PATH=/oxa
 OXA_TOOLS_PATH=$OXA_PATH/oxa-tools
-OXA_CONFIG_PATH=$OXA_PATH/configuration
+OXA_CONFIG_PATH=$OXA_PATH/edx-configuration
+OXA_ENV_SETTINGS_PATH=$OXA_PATH/oxa-tools-config
```

Comment:
@btelnes defines similar for oxa-tools-config in his PR, so you'll want to resolve.

see: https://github.com/Microsoft/oxa-tools/pull/13/files